lab1/Domareva/src/app/mycomponent/mycomponent.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component} from '@angular/core';
+
+@Component({
+  selector: 'app-mycomponent',
+  templateUrl: './mycomponent.component.html',
+  styleUrls: ['./mycomponent.component.css']
+})
+export class MycomponentComponent  {
+  text = '';
+  kot = 'meow';
```

Comment:
nice, but unusable variable

Reply:
it was the test variable, I forgot about it